tests/onegov/election_day/utils/test_archive_generator.py

```diff
@@ -0,0 +1,221 @@
+from functools import reduce
+from onegov.ballot import Vote, BallotResult
+from onegov.core.utils import module_path
+from onegov.election_day.models import Municipality, ArchivedResult
+from onegov.election_day.utils import add_local_results
+from onegov.election_day.utils.archive_generator import ArchiveGenerator
+from datetime import date
+from fs.walk import Walker
+from collections import Counter
+from fs.zipfs import ReadZipFS
+from fs.tempfs import TempFS
+from fs.copy import copy_dir
+from fs.osfs import OSFS
+
```

Comment:
General hint: The election day test suite is very slow, mostly because importing elections and votes take long, espcially when doing over and over again in different tests. That's why I recently started testing more functionality in one test (see e.g. test for election utils) so that import is done only once.

Think about merging some tests into one, e.g. testing query votes and elections, creating zip, checking zip etc. If you use an app without pre-added data you can also test how it behaves in case no given data is given.